content/docs/command-reference/import.md

```diff
@@ -65,7 +65,7 @@ path in the <abbr>workspace</abbr>. It records enough metadata about the
 imported data to enable DVC efficiently determining whether the local copy is
 out of date.
 
-To actually [track the data](/doc/tutorials/get-started/data-versioning),
+To actually [version the data](/doc/tutorials/get-started/data-versioning),
```

Comment:
not sure this is the best term ... but okay, let' keep it for now

Reply:
"To actually version the data, git add (and git commit) the import stage." seems accurate to me 🙂

Reply:
Plus the link goes to the Versioning use case.